account/src/main/java/com/hms/lib/commonmobileservices/account/google/GoogleAccountServiceImpl.kt

```diff
@@ -97,4 +99,31 @@ internal class GoogleAccountServiceImpl(private val context: Context, signInPara
         }
     }
 
+    override fun startSmsRetriver(context: Context): Work<Unit> {
+        val worker: Work<Unit> = Work()
+
+        mGoogleSmsRetriver.startSmsRetriever().addOnSuccessListener { worker.onSuccess(Unit) }
```

Comment:
You need to create broadcast receiver to get content of the SMS. We can remove this feature from this sprint and do it in the next sprint. You need to examine in detail usage of the services.